src/features/app/components/Form/Form.styles.js

```diff
@@ -42,8 +42,11 @@ export const FormContent = styled.form`
 `;
 
 export const HelpLink = styled(Link)`
-  color: ${(props) => props.theme.color.blue700};
+  margin-left: 0.25rem;
   margin-bottom: 0.25rem;
+  white-space: nowrap;
+  overflow: hidden;
```

Comment:
do we need this overflow: hidden? maybe we move the link to the next line if there is no space for it instead of hiding it?